lib/mailers/previews/casa_admin_mailer_preview.rb

```diff
@@ -1,11 +1,11 @@
 class CasaAdminMailerPreview < ActionMailer::Preview
   def account_setup
-    casa_admin = CasaAdmin.find_by(id: params[:id]) || CasaAdmin.last
+    casa_admin = params.has_key?(:id) ? CasaAdmin.find_by(id: params[:id]) : CasaAdmin.last
```

Comment:
It's weird because in practice it seems to me you'll end with the same behavior from the previous implementation. Considering the ticket itself asks for an error if the user is absent. :thinking:

Reply:
I understand from the PROBLEM that we need to show the error only if `id` is given and the user is not found. If the `id` is not provided, keep the behavior.

Reply:
What do you think about this @compwron?

Reply:
> show the error only if id is given and the user is not found. If the id is not provided, keep the behavior.

This is correct

Reply:
Ok